eng/packages/http-client-csharp/package.json

```diff
@@ -59,7 +50,17 @@
     "vitest": "^1.4.0",
     "copyfiles": "^2.4.1",
     "eslint": "^8.57.0",
+    "@eslint/js": "^9.2.0",
     "prettier": "~3.0.3",
     "typescript-eslint": "^7.8.0"
-  }
+  },
+  "azure-sdk/emitter-package-json-pinning": [
```

Comment:
given this is used in `eng\common\scripts\typespec\New-EmitterPackageJson.ps1`, how do we set the peer dependencies for local npm install?

Reply:
We still use devDependencies for pinning our peers or our dependencies' peers.   We're only changing the our model for describing which of our devDependencies need to be included in emitter-package.json's devDependencies